safe/definitionsv4/post_processors.py

```diff
@@ -184,20 +184,40 @@ def post_processor_affected_function(**kwargs):
     'description': tr(
         'Post processor to calculate the number of affected females'),
     'input': {
-        'population': {
-            'value': population_count_field,
-            'type': field_input_type
-        },
-        'gender_ratio': {
-            'value': female_ratio_field,
-            'type': field_input_type
-        }
+        # input as a list means, try to get the input from the
+        # listed source. Pick the first available
+        'population': [
+            {
+                'value': population_count_field,
+                'type': field_input_type,
+            },
+            {
+                'value': exposure_count_field,
+                'field_param': exposure_population['key'],
+                'type': dynamic_field_input_type,
+            }],
+        'gender_ratio': [{
+                'value': female_ratio_field,
+                'type': field_input_type
+            },
+            {
+                'type': keyword_input_type,
+                'value': [
+                    'inasafe_default_values',
```

Comment:
It should be covered in https://github.com/inasafe/inasafe/issues/3325#issuecomment-256316782